### PR TITLE
chore: migrate Pagination

### PR DESCRIPTION
**Pull request by anucreative**
## DESCRIPTION

Breaking changes:
- Removed `leftArrow` and `rightArrow` as not used in codebase : https://github.com/search?q=org%3AWTTJ+rightArrow&type=code

## SCREENSHOTS / SCREEN RECORDINGS

image

## COMPATIBILITY

- [ ] Tested on Safari (desktop)
- [x] Tested on Chrome (desktop)
- [ ] Tested on Firefox (desktop)
- [ ] Tested on mobile device sizes
- [ ] Tested on tablet device sizes
- [ ] Tested on IOS Safari (either device or simulator)

## QA

- [x] Thoroughly tested in local environment
- [ ] Added tests for all new features
- [ ] Added tests that considered edge cases

### CODE REVIEW

File: lib/src/components/Pagination/index.tsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
+import type { ComponentPropsWithRef } from 'react'
+import React, { forwardRef, useCallback, useRef } from 'react'
+
+import { classNames } from '@/utils'
+import { LeftIcon, RightIcon } from '@old/Icons'
+
+import styles from './pagination.module.scss'
+import { usePages } from './utils'
+
+export interface PaginationOptions {
```
Comment (Isaacmaamouche) [inline lib/src/components/Pagination/index.tsx]: We just need a types.ts file and we good ❤️

**Review by Isaacmaamouche — APPROVED** [review]
